.eslintrc

```diff
@@ -21,6 +21,6 @@
     }
   ],
   "rules": {
-    "no-console": "warn"
+    "no-console": ["warn", { "allow": ["warn", "error"]}]
```

Comment:
this is just a proposal so that `console.warn` and `console.error` aren't marked by the linter plugin since they are valid logs usually

please disregard this if you think it's not needed

Reply:
I think this is a good change.
the linter rule attempts to help the fact that occasionally `console.log` are left behind from debugging, but `console.warn` and `console.error` are rarely used in debugging

Good stuff